src/mixins/portfolio.js

```diff
@@ -214,6 +216,10 @@ export default {
         this.currentTab = 'collected';
       }
     },
+    async updateUserStats(address) {
+      const { data } = await this.$api.get(getUserNFTStats(address));
+      this.userStats = data;
+    },
```

Comment:
Why not put inside `<UserStatsBase>`?

Reply:
And add a watcher for `statWallet` to refresh the data

Reply:
For this pattern, the parent does not need to know the details of fetch & data structure, however, the parent can set `ref="userStats"` in `<UserStatsBase />`, then fetch the data using `this.$refs.userStats.fetch()`

Reply:
Maybe `<UserStatsBase/>` can rename to `<UserStatsController />`

Reply:
Indeed make sense.